colpivot.sql

```diff
@@ -103,8 +109,10 @@ create or replace function colpivot(
             foreach col in array key_cols loop
                 if i > 0 then
                     on_e := on_e || ' and ';
+                    on_e := on_e || '(' || ali || '.' || quote_ident(col) || ' = _key.' || quote_ident(col) || ' OR (' || ali || '.' || quote_ident(col) || ' IS NULL AND _key.' || quote_ident(col) || ' IS NULL)) ';
+                else
+                    on_e := on_e || ali || '.' || quote_ident(col) || ' = _key.' || quote_ident(col) || ' ';
```

Comment:
Beats me. I would have to test the function with and without checking for null. I'm also in the "fire and forget" mode 😆 But as I use it in the production, I guess it works regardless of maybe useless bloat.
I will check it out, if I have time for it.